### DISCUSSION TITLE
Backup all docker and CasaOS settings

### DISCUSSION THREAD

**Discussion by AlexFigas** [IceWhaleTech/CasaOS#655] [Ideas]
I'm looking forward for a backup functionality in CasaOS. Backup all dockers, all CasaOS settings everything. Then click restore button and boom everything is installed back and configured as before.
I change of OS's all the time so it would be a game changer for me.

**Comment by ksdfpr** [comment]
This is a good idea! What is the current situation? What is the best practice up to now?

**Comment by leesonaa** [comment]
I tried the following：

1.export all compose files
![image](https://github.com/IceWhaleTech/CasaOS/assets/97571961/67d4f730-98f6-4b7b-847e-61c30c6a203c)

2.tar appdata
![image](https://github.com/IceWhaleTech/CasaOS/assets/97571961/514f7add-0498-403a-aee3-01cfd47aa1af)
